lib/blockchain/common.js

```diff
@@ -99,7 +105,12 @@ function fetchAndInstall (coinRec) {
   es(`wget -q ${url}`)
   es(`tar -xzf ${downloadFile}`)
 
-  if (_.isEmpty(binaries.files)) {
+  if (binaries.build) {
```

Comment:
Seems this was only used for debugging and will be removed right?

Reply:
no, `binaries.build` is new "command", and we need this for building cardano, but if you mean that `console.log` a few lines below, then yes, this must be removed :)

Reply:
I am not sure whether `build` is good enough name. I wanted to propose `buildCmd` but that would suggest that the argument is a string which it isn't ... :-(

Reply:
```
  ADA: {
    url: 'https://github.com/input-output-hk/cardano-sl/archive/3.1.0.tar.gz',
    build: {folder: 'cardano-sl-3.1.0', script: 'cardano-build.sh'},
    dir: 'cardano-sl-3.1.0',
    files: [['cardano-testnet-wallet', 'cardano-node']]
  }
```

ok - when build will be using `dir` param, then we don't need `folder` and it can be `buildCmd` then.. do you agree?